### ISSUE TITLE
Need to increase value of property NestingDepth

### ISSUE THREAD

**Issue by davecarter** [scm-spain/frontend-pre-commit-rules#3]
### NestingDepth: 4 Proposal

In order to be able to **overwrite css** properties sometimes we need to create deeper nesting structures specially when we have:

```
.re-component {
   .sui-subcomponent {
      .a {
         &:hover {
           // hover states are an example;
         }
      }
   }
}
```

**Comment by nucliweb** [comment]
We can keep the current rule and use this structure

```
.sui-component {
   .sui-subcomponent {
      .a {
         // Styles...
      }
      .a:hover {
         // hover states are an example;
      }
   }
}
```

@nahuelsotelo, what you think?

**Comment by davecarter** [comment]
It works.. 
Looks like the `Grouping selectors linting rule` is not complaining.
But anyway. I'd rather increase just one step further to get a more readable SASS declarations structure.

Waitting for feedback from @nahuelsotelo and @nucliweb

**Comment by nahuelsotelo** [comment]
@davecarter @nucliweb 
Already out of nesting levels...:

```
.drv-Card {
  &-codeToggleBtn {
    &.is--open {
      &:before { ... }
      &:after { ... }
    }
  }
}
```

Compiled is not as big as it seems but is nicer to write it like this:

```
.drv-Card-codeToggleBtn.is--open:before { ... }
```

Just a pseudo element and a state class

**Comment by davecarter** [comment]
#### WE WANT :four: :arrow_right_hook: !!!

#### WE WANT :four: :arrow_right_hook: !!!

:smile:

**Comment by nahuelsotelo** [comment]
http://media.giphy.com/media/lZHsZkyWG7CEg/giphy.gif

**Comment by nahuelsotelo** [comment]
![](http://media.giphy.com/media/lZHsZkyWG7CEg/giphy.gif)
